### PR TITLE
Frank hemle flex

### PR DESCRIPTION
**Pull request by Frankzhac**
Fixing portfolio pg.

### CODE REVIEW

**Comment by aretoodeetoo** [comment]
**MVP:**
* [x] Review the [example resume](resume-example.png).  Notice how simplistic the example resume is.  Develop a simple layout of your choosing. If you are struggling to be creative, you may use the example resume as your design file. 
**Note: you are only required to build one page, anything more than that would be stretch.**
* [x] The resume content will be provided by you. The content can be about you or a fictional character.  
* [x] Content: Navigation - Build a simple navigation with 4 items of your choosing
* [x] Content: Intro - Have a short introduction as to why you would be a good hire
* [x] Content: Skills - Showcase a list of skills you have somewhere on your resume
* [x] Content: Work History - Showcase your work history somewhere on your resume
* [x] Content: Contact - Provide some way a potential employer could contact you.  Phone number, email, or a full on contact form (doesn't have to work)
* [x] Variables: Incorporate variables in your project for color and font stacks.  
* [x] Nesting: Every selector should be nested inside the main container.  Avoid having global styles unless they are element level.
* [x] Mixins: Create 2 mixins of your choosing. Hint: It's super helpful to use flexbox properties in mixins
* [x] Mobile: Use nested-at rules to provide a mobile version of your resume.  Use `500px` as a `max-width` for mobile.

**What Looks Good:**

- Frank, your resume page looks incredible. You did an amazing job on the styling. It's attention-grabbing, intuitive, and clearly shows off your skills, work history, and contact. Very well done.
- Love that you created a portfolio page and link to your GitHub to show off your code. Consider deploying those projects and linking to the webpage!
- Mobile looks good!

**Suggestions For Improvement:**
- When you do link out to your code, use `target="_blank"` in your `<a>` tags -- that will pop out your link into a new tab, which is a good user experience AND keeps the user on your resume page. 

**Project Rating:** 3 Stars 🌟 🌟 🌟

File: index.html [hunk 1]
```diff
@@ -52,81 +51,105 @@ <h1>Franck Le 🐱‍👤</h1>
...
+            <tr>
+              <th>Dates</th>
+              <th>Work</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <tr>
+              <td>2017-Present</td>
+              <td>Freelance Web designer</td>
```
Comment (christie-roy) [inline index.html]: Just for UI - be consistent with capitalization and capitalize Designer

File: portfolio.html [hunk 2]
```diff
@@ -53,24 +54,32 @@ <h1>Not so interesting stories</h1>
...
+        <p>Iterative approaches to corporate strategy foster collaborative<br>thinking to further the overall value proposition.</p>
         <button onclick="window.location.href = 'https://github.com/Frankzhac/UI-III-Flexbox/tree/master/great-idea';">View code on GitHub</button>
       </div>
 
       <div class="top-content">
         <img src="img/sprintChallenge.png" alt="strategy">
         <h2>Sprint Challenge User Interface</h2>
-        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <p>Iterative approaches to corporate strategy foster collaborative<br>thinking to further the overall value proposition.</p>
         <button onclick="window.location.href = 'https://github.com/Frankzhac/Sprint-Challenge--User-Interface';">View code on GitHub</button>
```
Comment (christie-roy) [inline portfolio.html]: Whoa!! Adding some onClick! Very impressive. See if you can make this link redirect to a new tab.

File: index.html (hunk 1 above)
Comment (aretoodeetoo) [inline index.html:108]: Just for UI - be consistent with capitalization. == `Designer`

File: portfolio.html [hunk 3]
```diff
@@ -53,24 +54,32 @@ <h1>Not so interesting stories</h1>
...
 
       </div>
 
-      <div class="bottom-content">
+      <div class="top-content">
         <img src="img/sweetBakery.png" alt="strategy">
         <h2>Sweet Eats Bakery</h2>
-        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <p>Iterative approaches to corporate strategy foster collaborative<br>thinking to further the overall value proposition.</p>
         <button onclick="window.location.href = 'https://github.com/Frankzhac/Sweet-Eats-Bakery';">View code on GitHub</button>
```
Comment (aretoodeetoo) [inline portfolio.html:73]: Very impressed with your use of an onClick element. See if you can add code to make this link pop out to a new tab.

File: less/index.less [hunk 4]
```diff
@@ -78,20 +206,21 @@ table {
...
+	display: flex;
+	flex-direction: row;
+	padding-top: 15px;
+	padding-bottom: 35px;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 2rem;
 }
 
 nav a {
```
Comment (aretoodeetoo) [inline less/index.less:226]: Your `nav a` should be nested inside your nav. There are a few nesting misses on this page so see if you can resolve those

**Review by aretoodeetoo — CHANGES_REQUESTED** [review]

**Pull request merged**